Cargo.toml

```diff
@@ -646,3 +646,6 @@ wasmi = { opt-level = 3 }
 x25519-dalek = { opt-level = 3 }
 yamux = { opt-level = 3 }
 zeroize = { opt-level = 3 }
+
+[patch."https://github.com/paritytech/litep2p"]
```

Comment:
Will be remove before merging once: https://github.com/paritytech/litep2p/pull/96, gets merged.